assets/js/components/settings/SettingsActiveModule/Header.js

```diff
@@ -98,13 +98,15 @@ export default function Header( { slug } ) {
 							/>
 							{ name }
 
-							{ slug === 'idea-hub' && (
+							{ [ 'idea-hub', 'thank-with-google' ].includes(
```

Comment:
It would be nice to DRY this list of experimental modules. How about we create `assets/js/components/settings/constants.js` and export an `EXPERIMENTAL_MODULES` constant from it, then use here and in `SetupModule.js`?